src/main/java/org/jitsi/videobridge/util/PartitionedByteBufferPool.java

```diff
@@ -260,7 +260,7 @@ void addStats(JSONObject stats)
             byte[] buf = pool.poll();
             if (buf == null)
             {
-                buf = new byte[DEFAULT_BUFFER_SIZE];
+                buf = new byte[Math.max(DEFAULT_BUFFER_SIZE, requiredSize)];
```

Comment:
was this actually happening?  isn't DEFAULT_BUFFER_SIZE 1500?

Reply:
Yeah, it's 1520, I'm not sure it was happening. We are observing some strange exceptions that may be explained by this, but I haven't been able to confirm.